### ISSUE TITLE
OTEL: ResourceExhausted error messages in DD

### ISSUE THREAD

**Issue by wesbillman** [TBD54566975/ftl#2239]
We are seeing logs of errors like this from OTEL in our DD logs.

```sh
OTEL: failed to upload metrics: rpc error: code = ResourceExhausted desc = grpc: received message larger than max (7735027 vs. 4194304)
```

**Comment by deniseli** [comment]
discussion: https://square.slack.com/archives/C07CHN37SKU/p1722535742804959

**Comment by safeer** [comment]
Fixed in #2242
